tasks/artembielov_task1/House price/model_v3.py

```diff
@@ -0,0 +1,58 @@
+import pandas as pd
+import numpy as np
+from sklearn.ensemble import RandomForestRegressor
+from sklearn.ensemble import GradientBoostingRegressor
+from sklearn.neural_network import MLPRegressor
+from sklearn.linear_model import Ridge
+from sklearn.linear_model import LinearRegression
+from mlxtend.regressor import StackingRegressor
+
+
+path = 'D:/projects/Python/Samsung2/House price/data/'

```

Comment:
Just for the future: 
try not to include paths, IPs, personal details on any other personalized or secret data into github commits 
This kind of data may unintentionally leak some secret information about a business if you working in a commercial project